common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4055,6 +4055,16 @@ object CelebornConf extends Logging {
       .checkValue(v => v >= 0.0, "Value must be no less than 0")
       .createWithDefault(0.2)
 
+  val CLIENT_PUSH_SORT_MAX_MEMORY_FACTOR: ConfigEntry[Double] =
+    buildConf("celeborn.client.spark.push.sort.memory.maxMemoryFactor")
+      .categories("client")
+      .doc("the max portion of executor memory which can be used for SortBasedWriter buffer (only" +
+        " valid when celeborn.client.spark.push.sort.memory.useAdaptiveThreshold is enabled")
```

Comment:
Maybe you can use ${CLIENT_PUSH_SORT_USE_ADAPTIVE_MEMORY_THRESHOLD.key} instead of hardcodig